nlp/admin/web/src/app/sentences-scroll/sentences-scroll.component.html

```diff
@@ -102,26 +101,26 @@ <h4 class="title"></h4>
                             [displayProbabilities]="displayProbabilities"
                             [displayStatus]="displayStatus"></tock-sentence-analysis>
 
-    <mat-table [hidden]="sentenceToUpdate" #table matSort [dataSource]="dataSource"
-               (matSortChange)="sortChange($event)">
-
-      <ng-container matColumnDef="select">
-        <mat-header-cell *matHeaderCellDef>
-          <mat-checkbox (change)="$event ? masterToggle() : null"
-                        [checked]="selection.hasValue() && isAllSelected()"
-                        [indeterminate]="selection.hasValue() && !isAllSelected()">
-          </mat-checkbox>
-        </mat-header-cell>
-        <mat-cell *matCellDef="let row">
-          <mat-checkbox (click)="$event.stopPropagation()"
-                        (change)="$event ? toggle(row) : null"
-                        [checked]="selection.isSelected(row)">
-          </mat-checkbox>
```

Comment:
Had to remove those ternary conditions because, in the new version, `$event` evaluates to true if and only if the checkbox is checked, causing unwanted behaviour.